lucien-ios/Dashboard/DashboardCollectionViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DashboardCollectionViewModel.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/13/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+class DashboardCollectionViewModel {
+
+    private var lendedComics = DashboardViewModel.dashboardData?.lending
+    private var borrowedComics = DashboardViewModel.dashboardData?.borrowing
+
+    func getComicCount() -> Int {
```

Comment:
If what you're returning is just a variable belonging to a private property in the view model, you should expose it in the form of a variable, not a function